util/sqlexec/utils_test.go

```diff
@@ -118,7 +112,7 @@ func (s *testUtilsSuite) TestEscapeSQL(c *C) {
 		err    string
 	}
 	time2, err := time.Parse("2006-01-02 15:04:05", "2018-01-23 04:03:05")
-	c.Assert(err, IsNil)
+	require.Nil(t, err)
```

Comment:
You can replace `require.Nil(t, err)` with `require.NoError(t, err)`

Reply:
fixed